rpcs3/main.cpp

```diff
@@ -408,23 +413,25 @@ int main(int argc, char** argv)
 
 	// Command line args
 	QCommandLineParser parser;
+	parser.setOptionsAfterPositionalArgumentsMode(QCommandLineParser::ParseAsPositionalArguments);
 	parser.setApplicationDescription("Welcome to RPCS3 command line.");
-	parser.addPositionalArgument("(S)ELF", "Path for directly executing a (S)ELF");
-	parser.addPositionalArgument("[Args...]", "Optional args for the executable");
 
 	const QCommandLineOption help_option    = parser.addHelpOption();
 	const QCommandLineOption version_option = parser.addVersionOption();
-	parser.addOption(QCommandLineOption(arg_headless, "Run RPCS3 in headless mode."));
-	parser.addOption(QCommandLineOption(arg_no_gui, "Run RPCS3 without its GUI."));
-	parser.addOption(QCommandLineOption(arg_high_dpi, "Enables Qt High Dpi Scaling.", "enabled", "1"));
-	parser.addOption(QCommandLineOption(arg_rounding, "Sets the Qt::HighDpiScaleFactorRoundingPolicy for values like 150% zoom.", "rounding", "4"));
-	parser.addOption(QCommandLineOption(arg_styles, "Lists the available styles."));
-	parser.addOption(QCommandLineOption(arg_style, "Loads a custom style.", "style", ""));
-	parser.addOption(QCommandLineOption(arg_stylesheet, "Loads a custom stylesheet.", "path", ""));
-	const QCommandLineOption config_option(arg_config, "Forces the emulator to use this configuration file.", "path", "");
-	parser.addOption(config_option);
-	parser.addOption(QCommandLineOption(arg_error, "For internal usage."));
-	parser.addOption(QCommandLineOption(arg_updating, "For internal usage."));
+	parser.addOption(QCommandLineOption(arg_headless, QCoreApplication::tr("CommandLine", "Run RPCS3 in headless mode.")));
```

Comment:
same borken code as above. and still no working translation at this point

Reply:
'borken code'?
Again.. the text will just display as-is if there is no translation loaded.

Reply:
Your code will always display "CommandLine" to the user instead of the existing text....

And why add translations to things that aren't actually translated by the software?
The translators will have to do extra work and will wonder how to test it

Reply:
And ditto for the wrong signature again.  If the translators don't want to translate it, they don't have to.
The plan would be to eventually have translations for these, so sooner or later they would ideally be translated.

Reply:
so how does the app now which language the user is on if it's randomly opened by a cli power user ?